packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts

```diff
@@ -64,18 +67,39 @@ export class AliasGatherer implements ParametersGatherer<Alias> {
       prompt: nls.localize('parameter_gatherer_enter_alias_name'),
       placeHolder: defaultAlias
     } as vscode.InputBoxOptions;
+    const expirationDays = {
+      prompt: nls.localize(
+        'parameter_gatherer_enter_scratch_org_expiration_days'
+      ),
+      placeHolder: defaultExpirationdate.toString()
+    } as vscode.InputBoxOptions;
     const alias = await vscode.window.showInputBox(aliasInputOptions);
     // Hitting enter with no alias will use the value of `defaultAlias`
     if (alias === undefined) {
       return { type: 'CANCEL' };
     }
-    return alias === ''
-      ? { type: 'CONTINUE', data: { alias: defaultAlias } }
-      : { type: 'CONTINUE', data: { alias } };
+    let scratchOrgExpirationInDays = await vscode.window.showInputBox(
+      expirationDays
+    );
+    if (
+      scratchOrgExpirationInDays === undefined ||
+      scratchOrgExpirationInDays === ''
+    ) {
+      scratchOrgExpirationInDays = defaultExpirationdate;
+    }
+    // return params;
```

Comment:
We should definitely be verifying the user input is a valid number in between the min/max number of days.